tc/autotuner/options_cache.h

```diff
@@ -163,9 +163,22 @@ struct OptionsCache {
       OptionsCacheKeyHash>
       store_;
 };
-} // namespace autotune
 
-std::string makeOptionsFilename(const std::string& fn);
+template <typename Backend>
+std::vector<typename Backend::MappingOptionsType> loadTopKFromCache(
+    const std::string& tc,
+    const std::string& entryPoint,
+    const std::string& cacheFileName,
+    const std::vector<const DLConstTensor*>& inputs,
+    const size_t numCandidates);
+
+template <typename Backend>
+void storeTopKToCache(
```

Comment:
some documentation?  the behavior of the function is not obvious from its name (see above)